javacord-core/src/main/java/org/javacord/core/util/gateway/DiscordWebSocketAdapter.java

```diff
@@ -703,6 +703,10 @@ private void sendIdentify(WebSocket websocket) {
                 .put("$device", "Javacord")
                 .put("$referrer", "")
                 .put("$referring_domain", "");
+        //TODO: If Intents are mandatory change this to add an empty string if no Intent is set
+        if (api.getIntent() != 0) {
+            data.put("intents", api.getIntent());
+        }
```

Comment:
Just set it directly. I would expect Discord to interpret `0` correctly as "no intents" (no intents = do not send any of the gateway events).
Can you please test this?
```suggestion
        data.put("intents", api.getIntent());
```

Reply:
Since we use as default all non privileged intents (from one of your requested changes) it will always have a valid number.